src/ui/components/NetworkMonitor/RequestDetails.tsx

```diff
@@ -99,6 +111,10 @@ export const TriangleToggle = ({ open }: { open: boolean }) => (
 );
 
 const parseCookie = (str: string): Record<string, string> => {
+  if (str.trim() === "") {
+    return {};
+  }
+
```

Comment:
Technically this function could still throw if it was passed a string like `";"` but I think that's a case where we'd _expect_ an error (that would be handled by a boundary).